compute_probs.py

```diff
@@ -131,7 +130,7 @@ def prefix_allowed_tokens_fn(batch_id, input_ids):
         early_stopping=True,
         return_dict_in_generate=True,
         output_scores=True,
-        max_length=200,
+        max_length=min(200, target_input_ids.shape[1] + 1),
```

Comment:
should it always be `target_input_ids.shape[1] + 1`, do we even need the 200?

Reply:
Good call, will update and re-run to verify

Reply:
Updated and confirmed it works